_includes/manuals/nightly/1.2_release_notes.md

```diff
@@ -12,6 +12,9 @@ Packages for Foreman and its various plug-ins are now being built for Ubuntu 20.
 
 Packages for Foreman and its various plug-ins that are built for CentOS 8 are now being tested on CentOS 8 Stream as well.
 
+#### Host registration improvements
+Redesigned form to generate the registration command with new fields `Install Packages` & `Repository & GPG keys`. New template kind `host_init_config` for host initial configuration (used to be `Linux registration default` template). This template is automatically assigned to every operating system. And added new API endpoint `/api/registration_commands` for generating registration command.
```

Comment:
It can be even more verbose
```suggestion
The host registration form has been redesigned and has new fields. `Install Packages` can be used to install additional packages while `Repository & GPG keys` can be used to configure additional software repositories for yum/dnf or apt.

A new template kind `host_init_config` for host initial configuration has been added to replace the old `Linux registration default` template. This template is automatically assigned to every operating system.

Additionally, a new API endpoint `/api/registration_commands` makes it easy to generate a registration command.
```

Reply:
I agree, the more verbose the better :) 

Is `Install Packages` a new field? 

If so i'd suggest "The `Install Packages` field can be used" and the same for the GPG. 

I would also say "A new `host_init_config` template replaces the old `Linux registration default` template. This new template is automatically assigned to every operating system. 

In general I am +1 to these suggestions!